pkg/cmd/pulumi/login.go

```diff
@@ -145,6 +159,9 @@ func newLoginCmd() *cobra.Command {
 	}
 
 	cmd.PersistentFlags().StringVarP(&cloudURL, "cloud-url", "c", "", "A cloud URL to log in to")
+	cmd.PersistentFlags().StringVar(&defaultOrg, "org-name", "", "A default org to associate with the login. "+
+		"Please note, this is only available in backends which support organizations, including the hosted and "+
```

Comment:
Should we error if this option is specified with a backend that doesn't support it?

Reply:
good idea!